acorn-loose/package.json

```diff
@@ -5,8 +5,15 @@
   "main": "dist/acorn-loose.js",
   "module": "dist/acorn-loose.mjs",
   "exports": {
-    "import": "./dist/acorn-loose.mjs",
-    "require": "./dist/acorn-loose.js"
+    ".": [
+      {
+        "import": "./dist/acorn-loose.mjs",
+        "require": "./dist/acorn-loose.js",
+        "default": "./dist/acorn-loose.js"
+      },
+      "./dist/acorn-loose.js"
```

Comment:
this is the fallback for node v13.0 and v13.1, which only handles the string form, not the object form.